### PR TITLE
[PSPE] Fixed chunk issues.

### PR DESCRIPTION
**Pull request by 7kasper**
WOOO

hack, but not so much extra memory or processing power

### CODE REVIEW

File: src/protocolsupport/listeners/PERequestListener.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import protocolsupport.zplatform.ServerPlatform;
+
+public class PERequestListener implements PluginMessageListener {
+
+	@Override
+	public synchronized void onPluginMessageReceived(String channel, Player player, byte[] message) {
+		if (channel.equals("PS|ReqChunk")) {
+			try {
+				DataInputStream in = new DataInputStream(new ByteArrayInputStream(message));
+				//System.out.println(in);
```
Comment (Shevchik) [inline src/protocolsupport/listeners/PERequestListener.java]: remove

File: src/protocolsupport/listeners/PERequestListener.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+				int x = in.readInt();
+				int z = in.readInt();
+				Chunk toUpdate = player.getWorld().getChunkAt(x, z);
+				System.out.println("Chunk requested: " + x + ", " + z);
```
Comment (Shevchik) [inline src/protocolsupport/listeners/PERequestListener.java]: remove

File: src/protocolsupport/listeners/PERequestListener.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import org.bukkit.plugin.messaging.PluginMessageListener;
+
+import protocolsupport.api.ProtocolSupportAPI;
+import protocolsupport.zplatform.ServerPlatform;
+
+public class PERequestListener implements PluginMessageListener {
+
+	@Override
+	public synchronized void onPluginMessageReceived(String channel, Player player, byte[] message) {
+		if (channel.equals("PS|ReqChunk")) {
```
Comment (Shevchik) [inline src/protocolsupport/listeners/PERequestListener.java]: extract the channel name to a public static variable

File: src/protocolsupport/ProtocolSupport.java [hunk 4]
```diff
@@ -133,6 +134,7 @@ public void onEnable() {
 		getServer().getPluginManager().registerEvents(new FeatureEmulation(), this);
 		getServer().getPluginManager().registerEvents(new ReloadCommandBlocker(), this);
 		getServer().getPluginManager().registerEvents(new MultiplePassengersRestrict(), this);
+		getServer().getMessenger().registerIncomingPluginChannel(this, "PS|ReqChunk", new PERequestListener());
```
Comment (Shevchik) [inline src/protocolsupport/ProtocolSupport.java]: here too

Comment (7kasper) [reply src/protocolsupport/ProtocolSupport.java]: Yeah I saw 😄

**Pull request merged**